components/stripe/sources/new-failed-invoice-payment/new-failed-invoice-payment.mjs

```diff
@@ -0,0 +1,18 @@
+import common from "../common/webhook-base.mjs";
+
+export default {
+  ...common,
+  key: "stripe-new-failed-invoice-payment",
+  name: "New Failed Invoice Payment",
```

Comment:
Invoice Payment Failed

Reply:
I think this one is ok actually -- "New Failed Invoice Payment" makes sense.